docker/packager/binary/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # rebuild in #33610
-# docker build -t clickhouse/binary-builder .
+# COV_TOKEN=xxx docker buildx build --secret id=COV_TOKEN -t clickhouse/binary-builder .
```

Comment:
We launch it from https://github.com/ClickHouse/ClickHouse/blob/master/tests/ci/docker_images_check.py#L236-L242, and I am afraid it requires much more adjustments